static/src/javascripts/projects/commercial/modules/dfp/define-slot.js

```diff
@@ -100,6 +113,11 @@ const defineSlot = (adSlotNode: Element, sizes: Object): Object => {
         slot.setTargeting('slot-fabric', fabricKeyValues.get(slotTarget));
     }
 
+    if (config.switches.adomik) {
+        slot.setTargeting('ad_group', adomikClassify());
+        slot.setTargeting('ad_h', new Date().getUTCHours().toString());
+    }
```

Comment:
They specifically wish to change the targeting for each slot. Why do you think that the targeting will be the same for all slots on the page? The random number is generated each time the function is called.